src/BUILD

```diff
@@ -572,6 +572,25 @@ cc_library(
     deps = ["//src:shared_lib"]
 )
 
+cc_binary(
+    name = "capi_benchmark",
+    srcs = [
+        "main_benchmark.cpp",
+    ],
+    linkopts = [
+        "-lxml2",
+        "-luuid",
+        "-lstdc++fs",
+        "-lcrypto",
+    ],
+    copts = [
+    ],
+    deps = [
+        "//src:ovms_lib",
```

Comment:
I don't think we link with libovms_shared.so here, therefore we do not benchmark actual lib